src/hotspot/os/bsd/os_bsd.cpp

```diff
@@ -972,6 +973,45 @@ bool os::dll_address_to_library_name(address addr, char* buf,
 // in case of error it checks if .dll/.so was built for the
 // same architecture as Hotspot is running on
 
+void *os::Bsd::dlopen_helper(const char *filename, int mode) {
```

Comment:
I thought BSD is switching to clang.

Reply:
What difference does it make if it does?

Reply:
I was trying to understand the BSD+gcc combination. We use clang on MacOS, so the only platform I thought would be affected would be one of the BSDs. 

But I thought those also moved to clang in their builds. Hence my question.

Reply:
These third-party libraries can come from anywhere. Given that the overhead added to `dlopen()` is almost immeasurably small, I think it's better to keep it.